assets/components/organisms/contact/contact-banner.twig

```diff
@@ -1,32 +1,30 @@
-<div class="bg-white p-4 p-md-5">
-  <div class="row">
-    <div class="col-md-6">
-      <h3>Contact</h3>
-      <p>Les guichets des « services aux étudiant-e-s » est le point de contact central pour traiter vos demandes.</p>
-      <div class="card card-body card-sm mb-2 flex-row flex-wrap justify-content-between justify-content-sm-start">
-        <div class="mr-3 w-sm-50">Lundi à jeudi</div>
-        <div><strong><time datetime="09:00">09:00</time> <span aria-label="à">›</span> <time datetime="18:00">18:00</time></strong></div>
-      </div>
-      <div class="card card-body card-sm mb-4 flex-row flex-wrap justify-content-between justify-content-sm-start">
-        <div class="mr-3 w-sm-50">Vendredi</div>
-        <div><strong><time datetime="09:30">09:30</time> <span aria-label="à">›</span> <time datetime="17:00">17:00</time></strong></div>
-      </div>
-      <p>
-        <a class="link-pretty" href="mailto:services.etudiants@epfl.ch">services.etudiants@epfl.ch</a>
-      </p>
-      <hr>
-      <p>
-        +41 21 693 43 45
-      </p>
-      <hr>
-      <address>
-        EPFL AA-DAF<br>
-        BP 1229 (bâtiment BP) Station 16<br>
-        CH - 1015 Lausanne
-      </address>
+<div class="row">
+  <div class="col-md-6">
```

Comment:
Why not using the `ContactPoint` schema for this block ?

Reply:
yep, added an issue for that #269